lib/utils.ts

```diff
@@ -123,3 +123,12 @@ export function getExePath(exeName: string, rootPath = rootPathDefault, exeExten
     return exeName
   }
 }
+
+/**
+ * Normalizes the given grammar scope so it always starts with `.`
+ * @param grammarScope such as 'source.python' or '.source.python'
+ * @returns the normalized grammarScope such as `.source.python`
+ */
+export function normalizeGrammarScope(grammarScope: string): string {
+  return grammarScope.includes(".") && grammarScope[0] !== "." ? `.${grammarScope}` : grammarScope
```

Comment:
My `normalizeGrammarScope` is a little different than what was being used here. We should determine the desired output and then normalize for that.

Reply:
Not sure why autocomplete-plug needs dots in the beginning. Probably we should add tests to prevent accidental breakage in the future.